rules/dropbox_rules/dropbox_linked_team_application_added.yml

```diff
@@ -0,0 +1,121 @@
+AnalysisType: rule
+Description: An application was linked to your Dropbox Account
+DisplayName: Dropbox Linked Team Application Added
+Enabled: false
+Filename: dropbox_linked_team_application_added.py
+Reference: Ensure that the application is valid and not malicious
+Runbook: Verify that this is expected
```

Comment:
is there an opportunity to be more descriptive about the runbook? Would it be appropriate to look at other actions taken by this user / directly contact the user / disable the user?

Reply:
Addressed in [`1297205` (#687)](https://github.com/panther-labs/panther-analysis/pull/687/commits/1297205451fb38f8c92a23625fda514f8c5bb864).